V4: can't resolve 'vm'

When upgrading from V3 (3.0.3) to V4 (4.0.1) I got the following error when running webpack in devmode: 

```
ERROR in ./node_modules/react-universal-component/dist/index.js
    Module not found: Error: Can't resolve 'vm' in '/Users/redacted/node_modules/react-universal-component/dist'
     @ ./node_modules/react-universal-component/dist/index.js 49:10-23
```

From what I can tell the issues stems from this line: https://github.com/faceyspacey/react-universal-component/blob/8e82a9df7c53bb5f33f4dc1cd8ea1b9d08a6c71b/src/index.js#L5

and its usage as a flow type here:
https://github.com/faceyspacey/react-universal-component/blob/8e82a9df7c53bb5f33f4dc1cd8ea1b9d08a6c71b/src/index.js#L116-L121

(whats more, `Context` does not seem to be a valid export from 'vm'? https://nodejs.org/docs/latest-v12.x/api/vm.html#. At least it has not been a mentioned export in any LTS version from node 4 to 12)

I will create a PR with a proposed fix shortly

--- a/src/index.js
+++ b/src/index.js
@@ -2,7 +2,6 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import hoist from 'hoist-non-react-statics'
-import { Context } from 'vm'
 import req from './requireUniversalModule'
 import type {
   Config,
@@ -117,7 +116,7 @@ export default function universal<Props: Props>(
       requireAsync: RequireAsync,
       props: Props,
       state: State,
-      context: Context,
+      context: Object = {},
       isMount?: boolean
     ) {
       if (!state.mod && loadingTransition) {

